torch/_dynamo/eval_frame.py

```diff
@@ -186,8 +186,9 @@ def __init__(self, mod: torch.nn.Module, dynamo_ctx):
 
     def _initialize(self):
         # Do this stuff in constructor to lower overhead slightly
-        if isinstance(self._orig_mod.forward, types.MethodType) and skipfiles.check(
-            inspect.getsourcefile(self._orig_mod.forward)
+        if (
+            isinstance(self._orig_mod.forward, types.MethodType)
+            and skipfiles.check(inspect.getsourcefile(self._orig_mod.forward)).skipped
```

Comment:
This seems like an error prone refactor, since you are now returning a tuple whereas previous it was returning a bool. Are you going to do something like disable `__bool__` on the ... tuple I guess to avoid bugs where you're on the old call site? Or you might consider keeping `skipfiles.check` at the old API and have a `check_verbose` or something.

Reply:
I am not going to audit that you got all the call sites

Reply:
Yea, this is a bit error prone, I can switch to a new name.